leaf-ui/rappid-extensions/AssignmentsTable.js

```diff
@@ -172,10 +174,10 @@ var AssignmentsTable = Backbone.View.extend({
     displayAbsoluteRelationshipAssignments: function(){
         this.model.getLinks().forEach(linkCell => {
             linkBbm = linkCell.get('link');
-            if(linkBbm.isValidAbsoluteRelationship()){
-                var linkRelationshipView = new LinkRelationshipView({model: linkBbm});
+            if(linkCell.isValidAbsoluteRelationship()){
+                var linkRelationshipView = new LinkRelationshipView({model: linkBbm, linkSrc: linkCell.getSourceElement().get('intention').get('nodeName'), linkDest: linkCell.getTargetElement().get('intention').get('nodeName')});
```

Comment:
And remove linkBbm variable from line 176 since we only use it once to pass in here
```suggestion
                var linkRelationshipView = new LinkRelationshipView({model: linkCell.get('link'), linkSrc: linkCell.getSourceElement().get('intention').get('nodeName'), linkDest: linkCell.getTargetElement().get('intention').get('nodeName')});
```